components/admin_console/admin_definition.jsx

```diff
@@ -2019,7 +2019,7 @@ const AdminDefinition = {
                         label: t('admin.team.editOthersPostsTitle'),
                         label_default: 'Allow Team Administrators to edit others posts:',
                         help_text: t('admin.team.editOthersPostsDesc'),
-                        help_text_default: 'When true, Team Administrators and System Administrators can edit other user\'s posts.  When false, only System Administrators can edit other user\'s posts.',
+                        help_text_default: 'When true, Team Administrators and System Administrators can edit other user\'s posts.  When false, only System Administrators can edit other user\'s posts. Regardless of selection Team Administrators and System Administrators can delete other user\'s posts.',
```

Comment:
```suggestion
                        help_text_default: 'When **true**, both Team Admins and System Admins can edit other users' posts.  When **false**, only System Admins can edit other users' posts. However, Team Admins and System Admins can always delete other users' posts.',
```

Reply:
@justinegeffen does markdown formatting actually work here? Doesn't ``'`` need to be escaped?

Reply:
I am not entirely sure about markdown formatting, but the ' definitely needs to be escaped.